.github/workflows/project-updater.yml

```diff
@@ -0,0 +1,23 @@
+name: Update projects when an issue/PR is created/labeled
+
+on:
+  issues:
+    types:
+      - opened
+      - labeled
+  pull_request:
+    types:
+      - opened
+      - labeled
+
+jobs:
+  add-to-project:
+    name: Add to the Release and Deferred blocker project
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/add-to-project@v0.1.0
+        with:
+          project-url: https://github.com/orgs/python/projects/2
+          github-token: ${{ secrets.ADD_TO_PROJECT_PAT }}
```

Comment:
Have you created a PAT and added it as a repo secret?

https://github.com/actions/add-to-project#inputs

Reply:
Yes, the PAT is already in place.  I haven't tested it though.